sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -4,9 +4,9 @@
 
```

Comment:
Oh, i don't have guts to review this file. It's nearly 5000 lines of change.

Reply:
I cannot make this any shorter.